### DISCUSSION TITLE
Store php in multiple different custom folders

### DISCUSSION THREAD

**Discussion by semivori** [hjbdev/pvm#23] [Ideas]
Is it possible to add the ability to store php in multiple different custom folders? This is important because I already have multiple versions of php installed and configured and don't want to install them again.

**Comment by MeowSky49887** [comment]
Currently, PVM doesn't support storing PHP in multiple custom folders by default. However, I am actively working on adding this feature, and it should be available by next month. Thank you for your patience!
